Containerfile.build

```diff
@@ -0,0 +1,69 @@
+# ---------------
+# Build
+# ---------------
+
+FROM maven:3.9.9-eclipse-temurin-21 AS builder
+
+WORKDIR /build
+
+COPY pom.xml settings.xml .
+RUN mvn -s settings.xml dependency:go-offline
```

Comment:
Do we want to make the `settings.xml` file a build arg to enable users to specify their own `settings.xml` file?